ci/gpu/build.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+# Copyright (c) 2018, NVIDIA CORPORATION.
+##############################################
+# Dask cuML GPU build and test script for CI #
+##############################################
+set -e
+
+# Logger function for build status output
+function logger() {
+  echo -e "\n>>>> $@\n"
+}
+
+# Set path and build parallel level
+export PATH=/conda/bin:/usr/local/cuda/bin:$PATH
+export PARALLEL_LEVEL=4
+
+# Set home to the job's workspace
+export HOME=$WORKSPACE
+
+################################################################################
+# SETUP - Check environment
+################################################################################
+
+logger "Check environment..."
+env
+
+logger "Check GPU usage..."
+nvidia-smi
+
+logger "Check versions..."
+python --version
+$CC --version
+$CXX --version
+conda list
+
+################################################################################
+# BUILD - Build libcuml and cuML from source
+################################################################################
+
+logger "Install cuml"
+
+conda install -c rapidsai -c rapidsai-nightly -c conda-forge -c numba -c rapidsai/label/cuda${CUDA_REL} -c nvidia/label/cuda${CUDA_REL} -c pytorch -c defaults cuml dask distributed cudf dask_cudf
```

Comment:
This project doesn't depend on CUDA right? So you should just use `-c rapidsai -c rapidsai-nightly`. You don't need the labels for the CUDA version.

Reply:
I'm confused by this. Do I need to use the pytorch and other conda channels in order to install cudf?

Reply:
cudf needs (https://github.com/rapidsai/cudf#conda):
- nvidia
- rapidsai
- conda-forge
- defaults

I'm suggesting adding rapidsai-nightly AFTER rapidsai. That allows for picking up the nightlies if nightly dask-cuml depends on that.